platform/languages.hpp

```diff
@@ -1,55 +0,0 @@
-#pragma once
-
-#include <array>
-#include <string>
-#include <utility>
-
-// The list of languages which can be used by TTS.
-// It shall be included in Android (jni) and iOS parts to get the languages list.
-// TODO: Now it is used only on iOS.
-// Manual sync with android/res/values/strings-tts.xml is needed.
-
-namespace routing::turns::sound
-{
-std::array<std::pair<std::string, std::string>, 38> const kLanguageList =
-{{
-  {"en", "English"},
```

Comment:
The second pair element was lost (the language name). Was it intentional? Was it used? What will be displayed in the UI?